exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/visitor/PrelVisitor.java

```diff
@@ -35,7 +38,9 @@
   public RETURN visitScan(ScanPrel prel, EXTRA value) throws EXCEP;
   public RETURN visitJoin(JoinPrel prel, EXTRA value) throws EXCEP;
   public RETURN visitProject(ProjectPrel prel, EXTRA value) throws EXCEP;
-
+  public RETURN visitHashToRandomExchange(HashToRandomExchangePrel prel, EXTRA value) throws EXCEP;
```

Comment:
Are 3 new methods necessary? Can `visitExchange` delegate to `prel` or use instance of?